index.js

```diff
@@ -50,8 +58,20 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
-
+function checkIfAnyMovieHasRating(movies,rate) {
+  if(!movies.length){
+    throw "Movies array is empty!";
+  }
+  let bool = movies.some((obj)=>{
+    if(obj.rated === rate){
+      return obj.rated;
+    }
+    if(!rate){
+      return obj.rated === "G";
```

Comment:
```suggestion
      rate ="G";
```
Although it's not causing tests to fail, this is actually incorrect. It's coincidentally passing the tests. if there's no rating it should default to "G". We can talk about it more 1:1 during office hours or class time!